### PR TITLE
Hookup to CMS addresses for footer

### PR DESCRIPTION
**Pull request by NathHorrigan**
This PR changes the hardcoded address for the version defined in the CMS

### CODE REVIEW

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -5,57 +5,28 @@ import PropTypes from 'prop-types'
...
         <li className={styles.footerAddress}>
-          <h2 className={styles.footerAddressTitle}>Glorious Oxfordshire</h2>
-          <p>
-            Unit 9<br />
-            Southill Business Park<br />
-            Charlbury<br />
-            OX7 3EW<br />
-            UK
-          </p>
+          <h2 className={styles.footerAddressTitle}>{offices.oxfordAddressTitle}</h2>
```
Comment (kaedroho) [inline src/components/footer/footer.js:13]: Maybe we should just make the API return a list of addresses? Might make adding/removing offices a little easier

**Review by kaedroho — APPROVED** [review]
LGTM. Just had thought but it doesn't matter much really

File: src/components/footer/footer.js (hunk 1 above)
Comment (NathHorrigan) [reply src/components/footer/footer.js:13]: I thought about this, But the GraphQL schema should return the exact interface shape of the model and I didn't want to spend too long updating the models 🤷

Comment (NathHorrigan) [reply src/components/footer/footer.js:13]: The whole site isn't getting overhauled in a couple of weeks so is probably worth addressing it then.

**Pull request closed without merging**